### PR TITLE
[GAWB-1826] Give users access to JES metadata for calls inside a workflow

### PR DESCRIPTION
**Pull request by rtitle**
JIRA:
https://broadinstitute.atlassian.net/browse/GAWB-1826

Related PRs:
https://github.com/broadinstitute/rawls/pull/694
https://github.com/broadinstitute/firecloud-ui/pull/785

Added `/api/workspaces/genomics/operations/{jobId}` as a pass-through to Rawls.

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [x] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala [hunk 1]
```diff
@@ -486,6 +489,15 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
...
     }
+
+    "Passthrough tests on the /workspaces/genomics/operations/%s path" - {
+      "OK status is returned for GET" in {
+        stubRawlsService(HttpMethods.GET, genomicsOperationsPath, OK)
+        Get(genomicsOperationsPath) ~> dummyUserIdHeaders("1234") ~> sealRoute(workspaceRoutes) ~> check {
+          status should equal (OK)
+        }
+      }
+    }
```
Comment (rushtong) [inline src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala:509]: Suggestion: For pass-throughs, we usually have tests to ensure we're NOT handling the http methods we do not want to pass-through, i.e. PUT, POST, etc., and expect the right method-not-handled error.

Comment (rtitle) [reply src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala:509]: OK thanks, will add.

Comment (rtitle) [reply src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceApiServiceSpec.scala:509]: Pushed commit with new tests.

**Comment by jmthibault79** [comment]
I suggest we pause review until the Rawls PR is approved, since it will affect this.

**Comment by rtitle** [comment]
Yes thanks, this PR will have to change per discussion in the Rawls pr.

**Comment by rtitle** [comment]
Should also be ready for re-review.

**Comment by jmthibault79** [comment]
👍

**Pull request merged**